Instruments/config/2018/Skimmer.cfg

```diff
@@ -153,41 +152,40 @@ file: WJetsToLNu_HT-2500ToInf.root
 
 [VV]
 merged_output: VV.root
-file_xs: 49.997 WWTo1L1Nu2Q_amcatnloFXFX.root WWToLNuQQ_NNPDF31.root
-file_xs: 51.723 WWTo4Q_NNPDF31.root
-file_xs: 5.52  WZTo2L2Q_amcatnloFXFX.root
-file_xs: 4.430  WZTo3LNu-amcatnloFXFX.root WZTo3LNu-amcatnloFXFX_ext1.root
-file_xs: 3.38 ZZTo2L2Q_amcatnloFXFX.root
-#file_xs: 0.637 WZTo3LNu_0Jets_MLL-50.root
-#file_xs: 0.337 WZTo3LNu_1Jets_MLL-50.root
-#file_xs: 0.066 WZTo3LNu_2Jets_MLL-50.root
-#file_xs: 0.096 WZTo3LNu_3Jets_MLL-50.root
-#file_xs: 75.80 WW.root
-#file_xs: 27.60 WZ.root
-#file_xs: 12.14 ZZ.root
+file_xs: 1.26	ZZTo4L.root
+file_xs: 0.564	ZZTo2L2Nu_ext2.root
+file_xs: 3.38	ZZTo2L2Q_amcatnloFXFX.root
+file_xs: 4.04	ZZTo2Q2Nu_amcatnloFXFX.root
+file_xs: 4.43	WZTo3LNu_ext1.root
+file_xs: 10.71	WZTo1L1Nu2Q_amcatnloFXFX.root
+file_xs: 3.033	WZTo1L3Nu_amcatnloFXFX.root
+file_xs: 5.52	WZTo2L2Q_amcatnloFXFX.root
+file_xs: 12.18	WWTo2L2Nu_NNPDF31.root
+file_xs: 49.997	WWToLNuQQ_NNPDF31.root
+file_xs: 51.72	WWTo4Q_NNPDF31.root
 
 [VVV]
 merged_output: VVV.root
-file_xs: 0.209  WWW_4F-amcatnlo_ext1.root
-file_xs: 0.057  WZZ-amcatnlo_ext1.root
-file_xs: 0.0147 ZZZ-amcatnlo_ext1.root
-file_xs: 0.0435 WZG-amcatnlo.root
+file_xs: 0.0147	ZZZ-amcatnlo_ext1.root
+file_xs: 0.057	WZZ-amcatnlo_ext1.root
+file_xs: 0.209	WWW_4F-amcatnlo_ext1.root
+file_xs: 0.168	WWZ-amcatnlo_ext1.root
 
 [VH]
 merged_output: VH.root
-file_xs: 0.054 ZHToTauTau_M125.root
-file_xs: 0.052 ZH_HToBB_ZToLL_M125.root ZH_HToBB_ZToLL_M125_ext1.root
-file_xs: 0.36 ZH_HToBB_ZToQQ_M125.root
-file_xs: 0.0526 WplusHToTauTau_M125.root
-file_xs: 0.0334 WminusHToTauTau_M125.root
-#file_xs: 0.072  ggZH_HToBB_ZToLL_M125.root
-#file_xs: 1.429  ggZH_HToBB_ZToNuNu_M125.root
+file_xs: 0.052	ZH_HToBB_ZToLL_M125.root ZH_HToBB_ZToLL_M125_ext1.root
+file_xs: 0.36	ZH_HToBB_ZToQQ_M125.root
+file_xs: 0.054	ZHToTauTau_M125.root
+file_xs: 0.0526	WplusHToTauTau_M125.root
+file_xs:0.0334	WminusHToTauTau_M125.root
+file_xs:0.072	ggZH_HToBB_ZToLL_M125.root
+file_xs:0.1227	ggZH_HToTauTau_ZToQQ_M125.root
```

Comment:
ggZH_HToBB_ZToLL_M125 and  ggZH_HToTauTau_ZToQQ_M125:
- XS is wrong
- don't use these samples